Gemfile

```diff
@@ -13,7 +15,7 @@ group :development, :test do
   gem "factory_bot_rails"
   gem "hirb"
   gem "pry-byebug"
-  gem "rspec-rails"
+  gem "rspec-rails", "3.7.1"
```

Comment:
I don't remember what issues we had with this version. What do you think about bumping it up to 3.7.2?

Reply:
I've bumped this over in my other branch, see: fbdf31c6d0762e1a0e5e2b2d5bf90e8bad94543e